bentoml/yatai/deployment/azure_functions/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG BENTOML_VERSION=latest
```

Comment:
does `pip install bentoml==latest` work?  might just remove the default value here